test/metabase/api/collection_test.clj

```diff
@@ -358,26 +359,78 @@
           (is (= [(default-item {:name "Dine & Dashboard", :description nil, :favorite false, :model "dashboard"})]
                  (mt/boolean-ids-and-timestamps
                   (:data (mt/user-http-request :rasta :get 200 (str "collection/" (u/the-id collection) "/items?archived=true")))))))))
-    (testing "Results include last edited information from the `Revision` table"
-      (mt/with-temp* [Collection [{collection-id :id} {:name "Collection with Items"}]
-                      User       [{user-id :id} {:first_name "Test" :last_name "User" :email "testuser@example.com"}]
-                      Card       [{card-id :id :as card}
-                                  {:name          "Card with history" :collection_id collection-id}]
-                      Revision   [_revision {:model    "Card"
-                                             :model_id card-id
-                                             :user_id  user-id
-                                             :object   (revision/serialize-instance card card-id card)}]]
-        (is (= [{:name "Card with history",
+    (mt/with-temp* [Collection [{collection-id :id} {:name "Collection with Items"}]
+                    User       [{user-id :id} {:first_name "Test" :last_name "User" :email "testuser@example.com"}]
+                    Card       [{card1-id :id :as card1}
+                                {:name "Card with history 1" :collection_id collection-id}]
+                    Card       [{card2-id :id :as card2}
+                                {:name "Card with history 2" :collection_id collection-id}]
+                    Card       [_ {:name "ZZ" :collection_id collection-id}]
+                    Card       [_ {:name "AA" :collection_id collection-id}]
+                    Revision   [_revision1 {:model    "Card"
+                                            :model_id card1-id
+                                            :user_id  user-id
+                                            :object   (revision/serialize-instance card1 card1-id card1)}]
+                    Revision   [_revision2 {:model    "Card"
+                                            :model_id card2-id
+                                            :user_id  user-id
+                                            :timestamp (time/from-now (time/days -2))
+                                            :object   (revision/serialize-instance card2 card2-id card2)}]]
+      ;; need different timestamps and Revision has a pre-update to throw as they aren't editable
+      (db/execute! {:update :revision
+                    ;; in the past
+                    :set {:timestamp (.minusHours (java.time.OffsetDateTime/now) 24)}
```

Comment:
I think you can use `java.time` to do `(t/minus (t/offset-date-time) (t/hours 24))` if you want to do this in Clojure instead of Java